development/docker-compose.mysql.yml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Can remove this file.

Reply:
I see, however rather keep those compose files. Drift management would recreate it each time there will be any change.

Reply:
They're just noise though for this app. Can we make drift manager smarter?

Reply:
For sure, depends on @cmsirbu as the PO. This is more about customizing the template, the drift management process is fine here.